src/snowflake/snowpark/column.py

```diff
@@ -644,7 +644,7 @@ def in_(
             ast = None
             if _emit_ast:
                 ast = proto.Expr()
```

Comment:
is the IR server-side branch for this already merged?

Reply:
Yup!